tests/acceptance/repo/settings-test.js

```diff
@@ -360,6 +360,31 @@ module('Acceptance | repo settings', function (hooks) {
     assert.dom('[data-test-settings-disabled-after-migration-modal]').exists();
   });
 
+  test('when repository is private and allow_config_imports setting is present', async function (assert) {
+    this.repository.private = true;
+    this.repository.createSetting({ name: 'allow_config_imports', value: true });
+
+    await settingsPage.visit({ organization: 'org-login', repo: 'repository-name' });
+
+    assert.ok(settingsPage.configImportsSection.exists, 'expected auto-cancellation section to exist');
+    assert.ok(settingsPage.allowConfigImports.isActive, 'expected allow_config_imports pushes to be present and enabled');
+
+    const settingToRequestBody = {};
+
+    server.patch(`/repo/${this.repository.id}/setting/:setting`, function (schema, request) {
+      settingToRequestBody[request.params.setting] = JSON.parse(request.requestBody);
+    });
```

Comment:
@weban could you please clarify a bit the purpose of this `patch` hook? I can't find any assertions against `settingToRequestBody` and the hook doesn't return anything.